models/contacts.js

```diff
@@ -23,37 +31,74 @@ const getContactById = async (contactId) => {
     console.log("getContactById:", error.message);
   }
 };
+*/
+const getContactById = async (contactId) => {
+  const data = await fs.readFile(contactsPath);
+    const contacts = JSON.parse(data);
+    const contact = contacts.find((contact) => contact.id === contactId);
+    return contact;
+}
 
-const removeContact = async (contactId) => {
+/*const addContact = async (name, email, phone) => {
   try {
     const data = await fs.readFile(contactsPath);
     const contacts = JSON.parse(data);
-    const contact = contacts.find((contact) => contact.id !== contactId);
-    console.log("Contact removed");
+    const newContactId = v4().replace(/-/g, "").substring(0, 20);
+    const newContact = { id: newContactId, name, email, phone };
+    contacts.push(newContact);
+    await fs.writeFile(contactsPath, JSON.stringify(contacts, null, 2));
+    console.log("Contact is added!");
   } catch (error) {
-    console.log("removeContact:", error.message);
+    console.log("addContact:", error.message);
   }
 };
+*/
 const addContact = async (name, email, phone) => {
-  try {
-    const data = await fs.readFile(contactsPath);
+  const data = await fs.readFile(contactsPath);
     const contacts = JSON.parse(data);
     const newContactId = v4().replace(/-/g, "").substring(0, 20);
     const newContact = { id: newContactId, name, email, phone };
     contacts.push(newContact);
     await fs.writeFile(contactsPath, JSON.stringify(contacts, null, 2));
     console.log("Contact is added!");
+    return contacts;
+
+}
+
+/*const removeContact = async (contactId) => {
+  try {
+    const data = await fs.readFile(contactsPath);
+    const contacts = JSON.parse(data);
+    const contact = contacts.find((contact) => contact.id !== contactId);
+    console.log(contact);
+    console.log("Contact removed");
   } catch (error) {
-    console.log("addContact:", error.message);
+    console.log("removeContact:", error.message);
   }
 };
+*/
+const removeContact = async (contactId) => {
+  const data = await fs.readFile(contactsPath);
+  const contacts = JSON.parse(data);
+  const contact = contacts.find((contact) => {
+    if(contact.id !== contactId){
+      console.log("Not found");
+    }else{
+      console.log("Contact removed!");
```

Comment:
Tu powinieneś zapisać na nowo, czyli to co masz w linii 62
await fs.writeFile(contactsPath, JSON.stringify(contacts, null, 2));